tutorials/plot_object_annotations.py

```diff
@@ -75,19 +78,25 @@
 mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp, color=color,
                     event_id=event_id)
 
-# Create some annotations specifying onset, duration and description
+###############################################################################
```

Comment:
I would not split those two snippets. The idea was to have a small summary of the other tutorials, and then dive into annotations and events.

Reply:
The advantage of two snippets is that the plotting layout is a little bit nicer in rendering. So I'm okay with this bit